pkg/storage/neonsan/api/api.go

```diff
@@ -44,12 +44,28 @@ type Volume struct {
 	ReplicationCount    int       `json:"replication_count"`
 	Status              string    `json:"status"`
 	MinReplicationCount int       `json:"min_replication_count"`
+	Role                string    `json:"role"`
+	Policy              string    `json:"policy"`
 	CreateTime          time.Time `json:"create_time" format:"ISO 8601"`
 	StatusTime          time.Time `json:"status_time" format:"ISO 8601"`
 	MetroReplica        string    `json:"metro_replica"`
+	ProvisionType       string    `json:"provision_type"` // thin or thick
+	MaxBs               int       `json:"max_bs"`
 	VolumeAllocated     int       `json:"volume_allocated"`
-	ProvisionType       string    `json:"provision_type"`
-	Role                string    `json:"role"`
+	RgName              string    `json:"rg_name"`
+	Encrypted           string    `json:"encrypted"`
+}
+
+type VolumeForClone struct {
+	ID                  int    `json:"id"`
+	Size                int    `json:"size"`
+	ReplicationCount    int    `json:"replication_count"`
+	MinReplicationCount int    `json:"min_replication_count"`
+	Role                string `json:"role"`
+	MaxBs               int    `json:"max_bs"`
+	Encrypte            string `json:"encrypte"`
+	KeyName             string `json:"key_name"`
+	Rg                  string `json:"rg"`
```

Comment:
What is RG meaning? Should it change to full name?

Reply:
I keep the parameter as same as the api document from of neonssan-api.